tests/test_github.py

```diff
@@ -61,6 +61,8 @@
 GITHUB_PULL_REQUEST_1_URL = GITHUB_PULL_REQUEST_URL + "/1"
 GITHUB_PULL_REQUEST_1_COMMENTS = GITHUB_PULL_REQUEST_1_URL + "/comments"
 GITHUB_PULL_REQUEST_1_COMMITS = GITHUB_PULL_REQUEST_1_URL + "/commits"
+GITHUB_PULL_REQUEST_1_REVIEWS = "https://api.github.com/repos/chaoss/grimoirelab-tutorial/pulls/12/reviews"
```

Comment:
I made the changes, but wouldn't the following still be there when Perceval fetches the data for a pull_request? : 
- in "_links": {"reviews": "<link to reviews>"}
- "reviews": <number of reviews for that pull request> 

Please let me know, @valeriocos! Thanks!

Reply:
@valeriocos, this is one of the testing problems I faced! Since the original pull request data (zhquan_example/repo) does not have reviews, there is no data at `zhquan_example/repo/pulls/\<num\>/reviews`. 

To counter this, and as you mentioned [here](https://github.com/chaoss/grimoirelab-perceval/issues/516#issuecomment-486258640), I used the reviews of another repository (which passed the test_pull_reviews test). 

But then, GITHUB_PULL_REQUEST_1_REVIEWS = GITHUB_PULL_REQUEST_1_URL + "/reviews" will not point correctly to the reviews data I have used.

@valeriocos, please clarify this. Thanks

Reply:
This comment may be answered with what written here: https://github.com/chaoss/grimoirelab-perceval/pull/518#pullrequestreview-231477830

Let me know if what is there solves your doubts

Reply:
> Let me know if what is there solves your doubts

It does, thanks!

Made the changes!